solr/CHANGES.txt

```diff
@@ -174,6 +174,8 @@ Bug Fixes
 
 * SOLR-16203: Properly initialize schema plugins loaded by SPI name (janhoy, hossman, Uwe Schindler)
 
+* SOLR-17074: Fixed not correctly escaped quote in bin/solr script (Dominique Béjean, Vincenzo D'Amore)
+
```

Comment:
Thanks @freedev for iterating on this pull request! I've moved the entry to the 9.5 section here and added attribution for both yourself and Dominique as reporter of the issue.

Please let us know if you'd prefer a different attribution e.g. see line 175 above, some folks use their handle and others their name.